### ISSUE TITLE
ink stitch is grayed out on mac OS X Catalina 10.15.5 

### ISSUE THREAD

**Issue by baggy-1291** [inkstitch/inkstitch#764]
I have tried reinstalling Inkscape 1.0 several times then installing ink stitch in Inkscapes "user\extension" directory as the lame directions states.  I have been chasing this for 2 days and no luck.  Whenever  I fire up Inkscape 1.0 and click on in the "extensions" in Inkscape, the "Ink/Stitch" choice is there and available.  Once that is selected, the "Commands" option is NOT grayed out, but the choices under "Commands" IS grayed out.
I tried installing any version of inkstich I could find, installed Inkscape 1.0 several times, all with the same results....grayed out and not selectable options with ink stitch!  Going nuts trying to find out how to get this thing to work....any suggestions on this delimma???
Thanks to all.....be safe

**Comment by Richard130948** [comment]
I have tried reinstalling Inkscape 1.0 several times then installing ink stitch in Inkscapes "user\extension" directory as the lame directions states. I have been chasing this for 2 days and no luck. Whenever I fire up Inkscape 1.0 and click on in the "extensions" in Inkscape, the "Ink/Stitch" choice is there and available. Once that is selected, the "Commands" option is NOT grayed out, but the choices under "Commands" IS grayed out.
I tried installing any version of inkstich I could find, installed Inkscape 1.0 several times, all with the same results....grayed out and not selectable options with ink stitch! Going nuts trying to find out how to get this thing to work....any suggestions on this delimma???
Thanks to all.....be safe

**Comment by CP0415** [comment]
This happened to me as well,
I opened the "User/extension" file and the I unloaded the "inkstitch-refs-tags-v1.28.0-osx-en_US (1)" directly into it but the menu was all grayed out.
I took all the files out of "inkstitch-refs-tags-v1.28.0-osx-en_US (1)" and put them directly into the extensions file instead of inside the original folder. I restarted the program and now the all the options are available.
Hope this helps.
